src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const [loading, setLoading] = useState(false);
```

Comment:
Consider renaming to `isLoading` to improve code style 

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig